lib/utils.pm

```diff
@@ -891,6 +892,29 @@ sub service_action {
     }
 }
 
+=head2 install_all_from_repo
+   will install all packages in repo defined by INSTALL_ALL_REPO variable
+   with ability to exclude some of them by using INSTALL_ALL_EXCEPT which suppose to contain
+   space separated list of packages
```

Comment:
yes, https://github.com/os-autoinst/os-autoinst/blob/master/t/05-pod.t might be a start.